Update commander.ts

CptHodor opened the pull request:
Removed:
- id:coordinated-strike-order

Added:
- id:coordinated-strike-pool

Changed:
-id coordinated-strike-base  now consumes id coordinated-strike-pool resource and has a "per_turn" frequency.

Message:
The Commander level 1 feature reads:
"Coordinated Strike! Gain the Coordinated Strike! Commander's Order." 

And the "Coordinated Strike" Order reads:
Coordinated Strike! (1/round) Free action:
you and an ally within 6 spaces both immediately make a weapon attack or cast a cantrip for free. You can do this INT times/Safe Rest.

So, I removed coordinated strikes from the commanders orders pool because you get the ability automatically and as far as I understand you cant pick the same order twice. I made the coordinate strike resource because the coordinated strike ability is both a per turn action and has a limited ammount of uses per rest based on the characters INT. I belive this is the best way to reflect that.

CptHodor committed:
Update commander.ts

Removed:
- id:coordinated-strike-order

Added:
- id:coordinated-strike-pool

Changed:
-id coordinated-strike-base  now consumes id coordinated-strike-pool resource and has a "per_turn" frequency.

Message:
The Commander level 1 feature reads:
"Coordinated Strike! Gain the Coordinated Strike! Commander's Order." 

And the "Coordinated Strike" Order reads:
Coordinated Strike! (1/round) Free action:
you and an ally within 6 spaces both immediately make a weapon attack or cast a cantrip for
free. You can do this INT times/Safe Rest.

So, I removed coordinated strikes from the commanders orders pool because you get the ability automatically and as far as I understand you cant pick the same order
twice. I made the coordinate strike resource because the coordinated strike ability is both a per turn action and has a limited ammount of uses per rest based on the characters INT. I belive this is the best way to reflect that.

martinsix commented on apps/web/data/classes/commander.ts:
```diff
@@ -221,15 +199,41 @@ const commanderFeatures: ClassFeature[] = [
+          },
+          maxValue: {
+            type: "formula",
+            expression: "1*INT",
```
you can just write `INT` here

martinsix commented on apps/web/data/classes/commander.ts:
```diff
@@ -221,15 +199,41 @@ const commanderFeatures: ClassFeature[] = [
+        id: "coordinated-strike-pool",
+        type: "resource",
+        resourceDefinition: {
+          id: "coordinated-strike-pool-0",
```
the `-0` suffix is usually used for the traits, not the resource. Maybe call the resource `coordinated-strikes` instead

martinsix commented on apps/web/data/classes/commander.ts:
```diff
@@ -221,15 +199,41 @@ const commanderFeatures: ClassFeature[] = [
-            "You and an ally within 6 spaces both immediately make a weapon attack or cast a cantrip for free.",
-          type: "action",
-          frequency: "per_safe_rest",
+          id: "coordinated-strike-base-0",
```
same thing here, the ability should just be called `coordinated-strike`

martinsix reviewed (commented):
Looks great! Just some small comments

CptHodor committed:
Update commander.ts

martinsix reviewed (approved):
looks great, thanks!

martinsix commented on apps/web/data/classes/commander.ts:
```diff
@@ -221,15 +199,41 @@ const commanderFeatures: ClassFeature[] = [
+          description: "You and an ally within 6 spaces both immediately make a weapon attack or cast a cantrip for free.",
+          resourceCost: {
+            type: "fixed",
+            resourceId: "coordinated-strike-pool-0",
```
oh sorry, this should now say `coordinated-strikes`

CptHodor committed:
Update commander.ts

Updated:
id:coordinated-strike-base ability now consumes resource id:coordinated -strikes.

CptHodor commented:
Testing the comment feature to see where this coment lands.

martinsix reviewed (approved):
looks good!
